src/doc/CreateHighlightDialog.js

```diff
@@ -79,8 +79,8 @@ class CreateHighlightDialog extends CreateAnnotationDialog {
     constructor(parentEl, config = {}) {
         super(parentEl, config);
 
-        this.allowHighlight = !!config.allowHighlight || true;
-        this.allowComment = !!config.allowComment || true;
+        this.allowHighlight = config.allowHighlight || false;
```

Comment:
Fixes bug where both highlight and comment buttons were displayed in the create dialog despite only plain highlights being enabled